packages/kbn-test/src/kbn_client/kbn_client_requester_error.ts

```diff
@@ -17,7 +17,9 @@ export class KbnClientRequesterError extends Error {
   }
 }
 function clean(error: Error): AxiosError {
-  const _ = AxiosError.from(error);
+  const _ = new AxiosError(error.message);
+  _.cause = error;
+  _.name = error.name;
   delete _.config;
   delete _.request;
   delete _.response;
```

Comment:
I don't know enough about the structure of axios errors, just want to double check:
Are we sure, we're still cleaning the needed bits here? Before, we built `_` from the whole error object and removed `config`, `request` and `response`. Now, `_` is built from `error.message` and the whole error object is added as `_.cause`. The removal happens from the bits that are in `error.message`, so just looking at the code I thought there might be a risk that `_.cause` exposes information that were deleted in the old implementation.

Reply:
I took this implementation from the axios `from` function, which was (unintentionally?) removed from 1.7.8 (see https://github.com/axios/axios/issues/6726). The good news is that they fixed this and I will just bump to 1.7.9, and we can return to our original implementation.

Reply:
Sounds good.
Nit: would be good to update the PR title and description to reflect the new target version

Reply:
@pheyos I took another look just to double check. `cause` has always been set to the incoming error in the `from` function (going back to v1.0.0). I think your concern is valid, and I wonder if should be deleting `cause` as well.

https://github.com/axios/axios/blob/b2cb45d5a533a5465c99559b16987e4d5fc08cbc/lib/core/AxiosError.js#L94C22-L94C27

```
AxiosError.from = (error, code, config, request, response, customProps) => {
  const axiosError = Object.create(prototype);

  utils.toFlatObject(error, axiosError, function filter(obj) {
    return obj !== Error.prototype;
  }, prop => {
    return prop !== 'isAxiosError';
  });

  AxiosError.call(axiosError, error.message, code, config, request, response);

  axiosError.cause = error;

  axiosError.name = error.name;

  customProps && Object.assign(axiosError, customProps);

  return axiosError;
};
```

cc @elastic/kibana-security

Reply:
Good catch, thanks for double checking!